libraries/standard/http/src/http_client.c

```diff
@@ -188,15 +415,31 @@ HTTPStatus_t HTTPClient_Send( const HTTPTransportInterface_t * pTransport,
     /* Send the body, which is at another location in memory. */
     if( returnStatus == HTTP_SUCCESS )
     {
-        returnStatus = _sendHttpBody( pTransport,
-                                      pRequestBodyBuf,
-                                      reqBodyBufLen );
+        if( pRequestBodyBuf != NULL )
+        {
+            returnStatus = _sendHttpBody( pTransport,
+                                          pRequestBodyBuf,
+                                          reqBodyBufLen );
+        }
+        else
+        {
+            IotLogDebug( "A request body was not sent: pRequestBodyBuf is NULL." );
```

Comment:
Suggest changing it to "No HTTP body in request sent: pRequestBodyBuf is NULL"

Reply:
I am not changing this. This means the same thing.